RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -77,6 +83,8 @@
 #   hcalLaserEventFilter *
 #  CSCTightHaloTrkMuUnvetoFilter *
 # CSCTightHalo2015Filter *
+   #globalTightHalo2016Filter*
```

Comment:
a comment with something like "uncomment this when filters become official" may be more useful than just plainly commented out